src/components/ha-markdown-element.ts

```diff
@@ -1,7 +1,12 @@
+import type { PropertyValues } from "lit";
 import { ReactiveElement } from "lit";
 import { customElement, property } from "lit/decorators";
+import hash from "object-hash";
 import { fireEvent } from "../common/dom/fire_event";
 import { renderMarkdown } from "../resources/render-markdown";
+import { CacheManager } from "../util/cache-manager";
+
+const markdownCache = new CacheManager<string>(1000);
```

Comment:
Is 1s enough?

Reply:
Yes few milliseconds should also be enough. Rebuilding a dashboard doesn't take too much time.